robyn/__init__.py

```diff
@@ -54,7 +54,9 @@ def __init__(self, file_object: str, config: Config = Config()) -> None:
         self.directories: List[Directory] = []
         self.event_handlers = {}
 
-    def _add_route(self, route_type, endpoint, handler, is_const=False):
+    def _add_route(
+        self, route_type: HttpMethod, endpoint: str, handler: Callable, is_const=False
+    ):
         """
```

Comment:
Adding types everywhere. Breaks my heart! 😭

Reply:
Sorry, I'm convinced this is the way to go with Python now :p My IDE gives me pretty useful hints with those